tests/integration/devfile/cmd_devfile_exec_test.go

```diff
@@ -25,25 +26,58 @@ var _ = Describe("odo devfile exec command tests", func() {
 		helper.CommonAfterEach(commonVar)
 	})
 
-	Context("When devfile exec command is executed", func() {
+	When("a component is created", func() {
 
-		It("should execute the given command successfully in the container", func() {
-			utils.ExecCommand(commonVar.Context, cmpName)
-			podName := commonVar.CliRunner.GetRunningPodNameByComponent(cmpName, commonVar.Project)
-			listDir := commonVar.CliRunner.ExecListDir(podName, commonVar.Project, "/projects")
-			Expect(listDir).To(ContainSubstring("blah.js"))
-		})
+		BeforeEach(func() {
+			args := []string{"create", "nodejs", cmpName, "--context", commonVar.Context}
+			helper.Cmd("odo", args...).ShouldPass()
+			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), commonVar.Context)
+			helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile.yaml"), filepath.Join(commonVar.Context, "devfile.yaml"))
```

Comment:
Do we need to copy this devfile? Won't the devfile created by running `odo create` be enough?

Reply:
It seems the PROJECTS_ROOT definition in the devfile.yaml is important. Some tests fail without copying this devfile.